modules/standard/SysError.chpl

```diff
@@ -32,6 +32,7 @@
    In the second form, a ``syserr`` is returned in an out error argument.
 
  */
+deprecated "The SysError module is deprecated. Please use 'OS' instead"
```

Comment:
Very minor point: Maybe put single quotes around 'SysError'.

Reply:
Thanks @jeremiah-corrado! Instead of `pragma "last resort"` I replaced the contents of `SysError` with `public use OS;`. I also added the quotes in the warning. I retested and it passed still. Look good?